puppet/modules/kubernetes/manifests/apiserver.pp

```diff
@@ -2,9 +2,9 @@
 class kubernetes::apiserver(
   $allow_privileged = true,
   Optional[Boolean] $audit_enabled = undef,
-  String $audit_log_directory = '/etc/kubernetes/audit',
-  String $audit_log_path = '/etc/kubernetes/audit/kubernetes-audit.log',
-  String $audit_policy_file = '/etc/kubernetes/audit/audit-policy.yaml',
+  String $audit_log_directory = '/var/log/kubernetes',
```

Comment:
This also might interfere with a default logrotate rule centos sets up. Worth checking how logrotate is setup by centos

Reply:
Looks like logrotate doesn't touch general files in `/var/log`
```
$ cat /etc/logrotate.conf | grep '/var/log'
/var/log/wtmp {
/var/log/btmp {
[root@luke-cluster|master|ip-10-99-53-8 ~]$ cat /etc/logrotate.d/* | grep '/var/log'
/var/log/chrony/*.log {
/var/log/puppetlabs/pxp-agent/*.log {
/var/log/cron
/var/log/maillog
/var/log/messages
/var/log/secure
/var/log/spooler
/var/log/wpa_supplicant.log {
/var/log/yum.log {
```

Reply:
Thanks